api/types/src/convert.rs

```diff
@@ -260,14 +258,6 @@ impl<'a, R: MoveResolver + ?Sized> MoveConverter<'a, R> {
                     transaction_payload,
                 })
             },
-
-            // Deprecated. Will be removed in the future.
-            ModuleBundle(modules) => TransactionPayload::ModuleBundlePayload(ModuleBundlePayload {
-                modules: modules
-                    .into_iter()
-                    .map(|module| MoveModuleBytecode::from(module).try_parse_abi())
-                    .collect::<Result<Vec<_>>>()?,
-            }),
         };
```

Comment:
Are there any transactions that use ModuleBundle as an output?  Including the code logic?

Reply:
No